beacon_node/network/src/sync/batch.rs

```diff
@@ -132,36 +115,36 @@ impl<E: EthSpec, B: BatchConfig> fmt::Display for BatchInfo<E, B> {
 
 #[derive(Display)]
 /// Current state of a batch
-pub enum BatchState<E: EthSpec> {
+pub enum BatchState<D: Hash> {
```

Comment:
@eserilev re-ordered the file to match unstable and parse the diff better. I found that you derived Debug on BatchState, which would dump all the blocks or columns to the command line. Re-added the custom impl of Debug on BatchState that we have on unstable.